vspace/src/main/webapp/WEB-INF/views/exhibition/space.html

```diff
@@ -172,14 +236,39 @@
 		    }
 		}
 		
+		function setCookie(name,value,days) {
+		    var expires = "";
+		    if (days) {
+		        var date = new Date();
+		        date.setTime(date.getTime() + (days*24*60*60*1000));
+		        expires = "; expires=" + date.toUTCString();
+		    }
+		    document.cookie = name + "=" + (value || "")  + expires + "; path=/";
+		}
+		
+		function getCookie(name) {
+		    var nameEQ = name + "=";
+		    var ca = document.cookie.split(';');
+		    for(var i=0;i < ca.length;i++) {
+		        var c = ca[i];
+		        while (c.charAt(0)==' ') c = c.substring(1,c.length);
+		        if (c.indexOf(nameEQ) == 0) return c.substring(nameEQ.length,c.length);
+		    }
+		    return null;
+		}
+		
 		$(window).on("load", function() {
+			var isResizeImgBtnClicked = getCookie('isResizeImgBtnClicked');
+			if(isResizeImgBtnClicked == "")
+				setCookie('isResizeImgBtnClicked', -1, null);
 		    let spaceHeight = $("#bgImage").css("height");
 		    $("#Module_1").css("height",spaceHeight);
 			$( window ).resize(function() {
 				$("#space a").remove();
 		    	drawLinks();
 		    });
 		    drawLinks();
+		    toggleImgSize(0);
```

Comment:
`toggleImgSize` expects a boolean, doesn't it. Pass true or false to make it easier to read/understand.